doc/patches/flutter/get-running-async-tasks.diff

```diff
@@ -0,0 +1,38 @@
+Enable re-entrance for `WidgetTester.runAsyncEnhanced`
```

Comment:
nit: maybe we can make it a `.md` file, and use "![image](https://github.com/fzyzcjy/flutter_convenient_test/assets/5236035/7c8655da-3345-4ae2-b12c-ebe8d3a28140)" to highlight the real diff

Reply:
well, this is a diff. and having this in a separate file (of the format produced by `git diff` and expected by `git apply`) allows the relatively simple workflow i suggest in the getting started section.

maybe a `patches/README.md` could help to motivate and explain, but i dont know what i would put in there except the explanation already contained in the `.diff`. The documentation is right next to the "code" in this way.

as for the format, [writing stuff before the `--- from-file ... +++ to-file ...` part seems to be the intended method](https://stackoverflow.com/a/20911434) of adding comments to `.diff`s

Reply:
Aha get it, I never know a `.diff` file can have "comments" like that!